src/stac_iitmandi/static/stac_iitmandi/css/style.css

```diff
@@ -11,7 +11,15 @@ li {
 }
 
 .bg-custom {
-    background-color: white;
+    background-color: #FFFFFF;
+    border-radius: 10px;
+}
```

Comment:
This class is useless, you can anyway use `bg-white` or `bg-light` to achieve the same.

Reply:
basically, i added that for boder-radius

Reply:
Have you tried `rounded-sm` class of bootstrap?

Reply:
Will try.